_summaries/glorot2010understanding_1.md

```diff
@@ -1,19 +1,46 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: {{glorot2010understanding}}
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: # slycane9
+score: # 7
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+Methods of training deep neural networks at the time (through back-propogation and random initialization) were not successful.  
+
+This paper explores alternative methods that can be used to better train these deep multi-layer networks.  Two of the main methods are the softsign activation function and normalized initialization.
+
+
+The researchers used a synethetic image dataset called Shapeset-3 x 2 to test their training configurations.
+
+
+They first tested the activation function in the model's hidden layers.  
+
+* Sigmoid
+	* Caused early saturation of top layer (impedes learning)
+	* Yielded the highest test error when compared to the other activation functions
+* Hyperbolic Tangent
+	* Causes layers to saturate one after the other (from first to last)
+	* Causes higher error rates than Softsign
+	* Gains the most improvement from using Normalized Initialization over Standard Initialization
+* Softsign
+	* Similar to Hyperbolic Tangent except it has polynomial asymptotes (causing a smoother curve)
+	* Its activation values follow the same trend at every layer and end at larger values
+	* Achieves the lowest error rate compared to the other two functions.
+
```

Comment:
@slycane9 I would expand on each of these a little more. In the paper they theorize why each of these preform differently and why saturation is happening in certain cases (especially sigmoid).